modules/library-ui-api/build.gradle.kts

```diff
@@ -25,6 +25,9 @@ android {
     defaultConfig {
         consumerProguardFiles("$projectDir/proguard-ui-api-consumer-rules.pro")
     }
+    buildFeatures {
```

Comment:
This is not needed on every module and, if it would, it should be added to the android-conventions gradle file